gokart/object_storage.py

```diff
@@ -67,11 +61,6 @@ def get_zip_client(file_path: str, temporary_directory: str) -> ZipClient:
         else:
             raise
 
-
     @staticmethod
     def is_readable_objectstorage_instance(file: object):
-        if isinstance(file, luigi.contrib.s3.ReadableS3File):
-            return True
-        else:
-            # for GCS object
-            return isinstance(file, FileWrapper)
```

Comment:
The bug was here.
This logic has prevented LocalFile from using `_LargeLocalFileReader`

Reply:
LGTM

Reply:
Great! Thanks!